Clinical-Genomics/scout
# Remove pileup

fix #1318

**How to test**:
1. Install this branch
1. Run scout and check that pileup is removed from variant and sv-variant page

**Expected outcome**:
One should only see IGV as viewing alternative now

**Review:**
